pkg/controller/nodeipam/ipam/cloud_cidr_allocator_test.go

```diff
@@ -215,6 +222,7 @@ func TestUpdateCIDRAllocation(t *testing.T) {
 					Name: "test",
 					NetworkInterfaces: []*compute.NetworkInterface{
 						{
+							Ipv6Address: "2001:db9::110",
```

Comment:
the test says `empty single stack node` , is this not making the node dual stack?

Reply:
Yes, you are correct. I changed this test back to single stack, and added a new one - so we have full coverage for the stack type of the node and the stack type of the cluster. Previously we did not have a test in which a single stack ipv4 cluster would be configured on a dual stack network. In such case we only want the ipv4 pod cidr. This is what I wanted to achieve originally.